source/menus/quest_status_menu.cpp

```diff
@@ -281,6 +353,62 @@ void QuestStatusMenu::draw() {
         dComIfGs_setPowerBracelets(new_power_bracelets_item_id);
         dComIfGs_setPowerBraceletsOwned(powerBraceletsIdToPowerBraceletsOwned(new_power_bracelets_item_id));
         break;
+    case MENU_ITEM_QUIVER:
+        new_arrows_capacity = dComIfGs_getArrowCapacity();
+        Cursor::moveListSimple(new_arrows_capacity);
+         if (new_arrows_capacity == NO_QUIVER - 1) {
+            new_arrows_capacity = NO_QUIVER;
+        } else if (new_arrows_capacity == NO_QUIVER + 1) {
+              new_arrows_capacity = ARROWS_30;
+        } else if (new_arrows_capacity == NO_QUIVER) {
+            new_arrows_capacity = NO_QUIVER;
+             } else if (new_arrows_capacity == ARROWS_30 - 1) {
+            new_arrows_capacity = NO_BOMBBAG;
+        } else if (new_arrows_capacity == ARROWS_30 + 1) {
+            new_arrows_capacity = ARROWS_60;
+         } else if (new_arrows_capacity == ARROWS_60 - 1) {
+            new_arrows_capacity = ARROWS_30;
+        } else if (new_arrows_capacity == ARROWS_60 + 1) {
+            new_arrows_capacity = ARROWS_99;
+        } else if (new_arrows_capacity == ARROWS_99 - 1) {
+            new_arrows_capacity = ARROWS_60;
+        } else if (new_arrows_capacity == ARROWS_99 + 1) {
+            new_arrows_capacity = ARROWS_99;
+            
+        } else {
+            new_arrows_capacity = dComIfGs_getArrowCapacity();
+        }
+        dComIfGs_setArrowNum(new_arrows_capacity);
+        dComIfGs_setArrowCapacity(new_arrows_capacity);
+        break;
+    case MENU_ITEM_BOMBAG:
+        new_bombs_capacity = dComIfGs_getBombCapacity();
+        Cursor::moveListSimple(new_bombs_capacity);
+         if (new_bombs_capacity == NO_BOMBBAG - 1) {
+            new_bombs_capacity = NO_BOMBBAG;
+        } else if (new_bombs_capacity == NO_BOMBBAG + 1) {
+              new_arrows_capacity = BOMBS_30;
+        } else if (new_bombs_capacity == NO_BOMBBAG) {
+            new_bombs_capacity = NO_BOMBBAG;

```

Comment:
This "else if" is redundant: if new_bombs_capacity is equal to NO_BOMBBAG, then set new_bombs_capacity equal to NO_BOMBBAG.

new_bombs_capacity is already equal to NO_BOMBBAG